html/script.js

```diff
@@ -56,15 +57,16 @@ const OpenMenu = (data) => {
 
     $(".main-wrapper").html(form.join(" "));
 
-    $("#qb-input-form").on("change", function (event) {
+    $(formID).on("change", function (event) {
         formInputs[$(event.target).attr("name")] = $(event.target).val();
     });
 
-    $("#qb-input-form").on("submit", async function (event) {
+    $("#" + formID).on("submit", async function (event) {
+		
         if (event != null) {
             event.preventDefault();
         }
-        let formData = $("#qb-input-form").serializeArray();
+        let formData = $("#" + formID).serializeArray();
```

Comment:
please try and stick to string literal rather than string concats:
$(\`#${formID}\`)